bin/init_helm.sh

```diff
@@ -75,13 +75,5 @@ export ISTIO_OUT=${ISTIO_OUT:-${ISTIO_BIN}}
 
 # install helm if not present, it must be the local version.
 if [ ! -f "${ISTIO_OUT}/version.helm.${HELM_VER}" ] ; then
-    TD=$(mktemp -d)
-    # Install helm. Please keep it in sync with .circleci
-    cd "${TD}" && \
-        ${DOWNLOAD_COMMAND} "${TD}/helm.tgz" "https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VER}-${LOCAL_OS}-amd64.tar.gz" && \
-        tar xfz helm.tgz && \
-        mv ${LOCAL_OS}-amd64/helm "${ISTIO_OUT}/helm-${HELM_VER}" && \
-        cp "${ISTIO_OUT}/helm-${HELM_VER}" "${ISTIO_OUT}/helm" && \
-        rm -rf "${TD}" && \
-        touch "${ISTIO_OUT}/version.helm.${HELM_VER}"
```

Comment:
Why did you remove downloading helm?

Reply:
the download failed- it doesn't matter anyway, because we pre-install helm in our builder image